src/components/navigation/Navbar.scss

```
@@ -0,0 +1,148 @@
+@import '../../constants.scss';
```

Prefer double quotes (`"`)
```suggestion
@import "../../constants.scss";
```

---

Done